azure/common/deploy-stage.yml

```diff
@@ -105,9 +115,29 @@ stages:
                       - ${{ each config_id in parameters.config_ids }}:
                           - ${{ config_id }}
 
+                - ${{ if parameters.notify }}:
+                  - bash: |
+                      export COMMIT_SHA=`echo $(Build.SourceVersionMessage) | cut -d' ' -f2`
+                      echo "##vso[task.setvariable variable=COMMIT_SHA]$COMMIT_SHA"
+                    displayName: Set COMMIT_SHA
+
+                  - template: '../components/update-github-status.yml'
+                    parameters:
+                      state: pending
+                      description: "Deploy to ${{ parameters.environment }} started"
+
+                - template: "../components/expose-pipeline-values.yml"
```

Comment:
We might want to review this approach later on (wondering if there is a nicer way to do it), but for now this is great